docs/node/getting-started.md

```diff
@@ -45,7 +45,7 @@ a stripped-down implementation of [ELIZA](https://en.wikipedia.org/wiki/ELIZA),
 the famous natural language processing program.
 
 ```bash
-$ mkdir -p proto && touch proto/eliza.proto
+$ mkdir -p connectrpc/eliza/v1 && touch connectrpc/eliza/v1/eliza.proto
```

Comment:
Specified that when the package is set to connectrpc.eliza.v1, the directory connectrpc/eliza/v1 must exist.

```
proto/eliza.proto:3:1:Files with package "connectrpc.eliza.v1" must be within a directory "connectrpc/eliza/v1" relative to root but were in directory "proto".
```